xmake/core/main.lua

```diff
@@ -74,6 +85,11 @@ function main._help()
             utils.cprint(menu.copyright)
         end
 
+        -- print features
+        if menu.features then
```

Comment:
Or you can write a `features.lua` script in `xmake/plugins/lua/scripts` and run `xmake lua features` to dump all supported features.